### PR TITLE
Adding Lintly parser for trivy

### PR DESCRIPTION
**Pull request by nandusekarv10**
Adding another lintly parser `Trivy`. It can scan docker image, repo and file systems. Currently in readme I have mentioned instructions  running it in fs mode. But parser will work even if you change arguments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,11 @@ Now you will see a review with linting errors...
     $ terrascan scan -d path/to/terraform/file -o json |lintly --format=terrascan
     ```
 
+-  [trivy](https://github.com/aquasecurity/trivy)
+    ```
+    $ trivy --quiet fs -f json path/to/directory/ |lintly --format=trivy
```
Comment (sarahc23) [inline README.md:75]: Mind fixing this and the two above here? It's a nitpick but it'd be more readable if you add a space between the pipe and `lintly` (i.e., `... path/to/directory/ | lintly ...`)

**Review by sarahc23 — APPROVED** [review]
Nice, looks great!

File: README.md (hunk 1 above)
Comment (scriptsrc) [reply README.md:75]: ha! This has been getting on my OCD too!

File: lintly/parsers.py [hunk 2]
```diff
@@ -470,6 +470,62 @@ def parse_violations(self, output):
...
+
+        json_data = json.loads(output)
+
+        violations = collections.defaultdict(list)
+
+        for data in json_data:
+            for violation_json in data["Vulnerabilities"]:
+                violation = Violation(
+                    line=0,
+                    column=0,
```
Comment (scriptsrc) [inline lintly/parsers.py:516]: Oh interesting - so trivy doesn't give us line numbers?

**Review by scriptsrc — APPROVED** [review]
Approving. Great job!

**Pull request merged**